src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -940,7 +941,7 @@ public ActivityIdFormat IdFormat
         private partial class KeyValueListNode
         {
             public KeyValuePair<string, string> keyValue;
```

Comment:
This should be `KeyValuePair<string?, string?>` right?

Reply:
right